src/musikcube/data/locales/zh_CN.json

```diff
@@ -1,7 +1,7 @@
 {
     "schemaVersion": 1,
     "strings": {
-        "default_overlay_title": "musikcube",
+        "default_overlay_title": "Musikcube",
```

Comment:
`musikcube` is actually branded completely lowercase. We should change the uppercase `M` back to `m` here and the few other places it was updated. :)